modular-docs-manual/content/topics/whats-new.adoc

```diff
@@ -0,0 +1,17 @@
+[id="whats-new_{context}"]
+= What's new
+
+The following list summarizes significant changes to the mod docs standard over time.
+Apply these changes when you create and revise content.
```

Comment:
```suggestion
Apply these changes only when you create and revise content.
```

Added only so the next sentence makes sense, but am open to other suggestions.

Reply:
Thanks for the suggestion, @ncbaratta. Updated.